modules/apps/frontend-js/frontend-js-components-web/src/main/resources/META-INF/resources/beta_indicator/BetaButton.tsx

```diff
@@ -30,7 +33,7 @@ export default function BetaButton({
 		<LearnResourcesContext.Provider value={learnResourceContext}>
 			<ClayTooltipProvider>
 				<div
-					className="tooltip-container"
+					className={classNames('tooltip-container', className)}
```

Comment:
Since you are adding the className to the "container" of the button, I think I would call this `containerClassName`. Because we call this component `*Button`, I would expect a `className` prop to be applied to the button element, not the container.